static/swagger/api_v3.json

```diff
@@ -62,16 +62,17 @@
       "in": "path",
       "description": "Page number of results to return, zero-indexed",
       "required": true,
-      "type": "number",
-      "format": "integer"
+      "type": "integer",
+      "minimum": 0
     },
     "year": {
       "name": "year",
       "in": "path",
       "description": "Competition Year (or Season). Must be 4 digits.",
       "required": true,
-      "type": "number",
-      "format": "integer"
+      "type": "integer",
+      "minimum": 1997,
+      "maximum": 9999
```

Comment:
Hmmm, I guess we do assume this is a 4 digit number in a lot of places, but this still feels clunky

Reply:
It only stops people from inputting a 5 digit number on the site and has zero effect on the actual request.
Also since the site is not showing the minimum I can also lower the minimum to 1000 in order for it to show a red border on an incorrect number of digits only if you want

Reply:
I don't think the UI being accurate is all that important of a concern, if people put in an invalid year it'll simply return no events. I think it'll be fine to just remove the minimum and maximum values here